src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -200,14 +188,14 @@ export const DownloadConfirmation: React.FunctionComponent<DownloadConfirmationP
     if (isSuccess && entityChildrenData) {
       updateStats(entityChildrenData.totalChildCount, entityChildrenData.sumFileSizesBytes)
     }
-  }, [updateStats, queryOrFolder, isSuccess, entityChildrenData])
+  }, [updateStats, folderId, isSuccess, entityChildrenData])
```

Comment:
This effect is specific to the folder (to get statistics about the children of the container)